crates/extensions/c8y_log_manager/src/lib.rs

```diff
@@ -58,10 +53,16 @@ impl LogManagerBuilder {
     }
 
     /// Connect this config manager instance to some mqtt connection provider
-    pub fn with_mqtt_connection(&mut self, mqtt: &mut MqttActorBuilder) -> Result<(), LinkError> {
+    pub fn with_mqtt_connection(
```

Comment:
We could consider replacing this with the `ServiceConsumer::set_connection` API to keep it consistent with other actors.

Reply:
Done